packages/dds/tree/src/test/exhaustiveRebaserUtils.ts

```diff
@@ -162,7 +162,7 @@ export function* generatePossibleSequenceOfEdits<TContent, TChangeset>(
 		initialState,
 		generateChildStates,
 		numberOfEdits,
-		(intention: number) => `${tagPrefix}${intention}` as RevisionTag,
+		(intention: number) => intention as RevisionTag,
```

Comment:
is there a reason we need to do this? Or is it just to typecheck?

Prefixing tags with different values actually is meaningful for rebasing logic. The analogous behavior would be to pass in some kind of baseline id for the different sessions, not just use the intention as-is.

I think none of the tests using this actually care if RevisionTag is a number/string/whatever. So we can probably get away with keeping it as a string.

Reply:
Pretty sure I did this in haste trying to get a build working. It can definitely go back to the way it was if these tests don't care